apps/docs/docs/reference/ui/molecules/DiscountBadge.mdx

```diff
@@ -0,0 +1,139 @@
+---
+description: A custom `Badge` that display product's discounts.
+keywords:
+  - UI library
+  - ecommerce
+  - react
+sidebar_custom_props:
+  image: https://vtexhelp.vtexassets.com/assets/docs/src/Badge___bcedffeb307527c1606be224a7fb5ac1.png
```

Comment:
@carolinamenezes  I think we can leave this image for the `DiscountBadge` preview. And change the `Badge` image for a simple label (e.g.: New arrival)